appserver/common/container-common/pom.xml

```diff
@@ -149,6 +149,18 @@
             <groupId>org.glassfish.ha</groupId>
             <artifactId>ha-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>fish.payara.payara-modules</groupId>
+            <artifactId>hazelcast-bootstrap</artifactId>
+            <version>${project.version}</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```

Comment:
I thought that no Lombok is to be included

Reply:
See internal hipchat for tnis